### DISCUSSION TITLE
New Source: Unleash [low-code CDK]

### DISCUSSION THREAD

**Discussion by arnaudselva** [airbytehq/airbyte#36101] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination?
Source : [Unleash API](https://docs.getunleash.io/reference/api/legacy/unleash)
Destination : AWS Redshift

* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB?
No

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
[Unleash](https://github.com/Unleash/unleash) is a popular open source feature management product. We are about to release it to our stack to be able to easily and reliably run A/B tests.
We would like to be able to extract the features running for any given date so that we can make some analysis on it.

* How often do you want to run syncs?
Once a day

* If this is an API source connector, which entities/endpoints do you need supported?
We only need to fetch the feature toggles via this dedicated endpoint : `http://unleash.host.com/api/client/features`
But I'd be happy to extend it to other endpoints as a future iteration.

## Describe the alternative you are considering or using
* What are you considering doing if you don’t have this integration through Airbyte?
I would create an Airflow task for it

## Are you willing to submit a PR?
Yes, happily, I'd love to try :)

**Comment by sh4sh** [comment]
Very excited to see this connector implemented with the low-code CDK! Let us know how we can help!

[a comment by an automated account is not shown]
